### PR TITLE
Few fixes for reference_appendices/faq.html

### PR DESCRIPTION
**Pull request by bmillemathias**
##### SUMMARY
formatting using `` instead of `
Add line break for long line
Rephrase or remove useless text.
Move some text.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
doc

### CODE REVIEW

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 1]
```diff
@@ -51,25 +51,24 @@ See the rest of the documentation for more information about how to organize var
 How do I get ansible to reuse connections, enable Kerberized SSH, or have Ansible pay attention to my local SSH config file?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Switch your default connection type in the configuration file to 'ssh', or use '-c ssh' to use
-Native OpenSSH for connections instead of the python paramiko library.  In Ansible 1.2.1 and later, 'ssh' will be used
+Switch your default connection type in the configuration file to ``ssh``, or use ``-c ssh`` to use
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:54]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 2]
```diff
@@ -51,25 +51,24 @@ See the rest of the documentation for more information about how to organize var
...
+Native OpenSSH for connections instead of the python paramiko library. In Ansible 1.2.1 and later, ``ssh`` will be used
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:55]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 3]
```diff
@@ -51,25 +51,24 @@ See the rest of the documentation for more information about how to organize var
...
 by default if OpenSSH is new enough to support ControlPersist as an option.
 
-Paramiko is great for starting out, but the OpenSSH type offers many advanced options.  You will want to run Ansible
-from a machine new enough to support ControlPersist, if you are using this connection type.  You can still manage
-older clients.  If you are using RHEL 6, CentOS 6, SLES 10 or SLES 11 the version of OpenSSH is still a bit old, so
+Paramiko is great for starting out, but the OpenSSH type offers many advanced options. You will want to run Ansible
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:58]: Removed double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 4]
```diff
@@ -51,25 +51,24 @@ See the rest of the documentation for more information about how to organize var
...
+from a machine new enough to support ControlPersist, if you are using this connection type. You can still manage
+older clients. If you are using RHEL 6, CentOS 6, SLES 10 or SLES 11 the version of OpenSSH is still a bit old, so
 consider managing from a Fedora or openSUSE client even though you are managing older nodes, or just use paramiko.
 
-We keep paramiko as the default as if you are first installing Ansible on an EL box, it offers a better experience
-for new users.
+We keep paramiko as the default as if you are first installing Ansible on these enterprise operating systems, it offers a better experience for new users.
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:63]: Good doc don't use abbreviation that are unknown

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 5]
```diff
@@ -51,25 +51,24 @@ See the rest of the documentation for more information about how to organize var
...
 
 .. _use_ssh_jump_hosts:
 
 How do I configure a jump host to access servers that I have no direct access to?
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-You can set a `ProxyCommand` in the
-`ansible_ssh_common_args` inventory variable. Any arguments specified in
+You can set a ``ProxyCommand`` in the
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:70]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 6]
```diff
@@ -84,32 +83,35 @@ You can create `group_vars/gatewayed.yml` with the following contents::
     ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q user@gateway.example.com"'
 
 Ansible will append these arguments to the command line when trying to
-connect to any hosts in the group `gatewayed`. (These arguments are used
-in addition to any `ssh_args` from `ansible.cfg`, so you do not need to
-repeat global `ControlPersist` settings in `ansible_ssh_common_args`.)
+connect to any hosts in the group ``gatewayed``. (These arguments are used
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:86]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 7]
```diff
@@ -84,32 +83,35 @@ You can create `group_vars/gatewayed.yml` with the following contents::
...
+in addition to any ``ssh_args`` from ``ansible.cfg``, so you do not need to
+repeat global ``ControlPersist`` settings in ``ansible_ssh_common_args``.)
 
-Note that `ssh -W` is available only with OpenSSH 5.4 or later. With
-older versions, it's necessary to execute `nc %h:%p` or some equivalent
+Note that ``ssh -W`` is available only with OpenSSH 5.4 or later. With
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:90]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 8]
```diff
@@ -84,32 +83,35 @@ You can create `group_vars/gatewayed.yml` with the following contents::
...
+older versions, it's necessary to execute ``nc %h:%p`` or some equivalent
 command on the bastion host.
 
 With earlier versions of Ansible, it was necessary to configure a
-suitable `ProxyCommand` for one or more hosts in `~/.ssh/config`,
-or globally by setting `ssh_args` in `ansible.cfg`.
+suitable ``ProxyCommand`` for one or more hosts in ``~/.ssh/config``,
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:95]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 9]
```diff
@@ -84,32 +83,35 @@ You can create `group_vars/gatewayed.yml` with the following contents::
...
+or globally by setting ``ssh_args`` in ``ansible.cfg``.
 
 .. _ssh_serveraliveinterval:
 
 How do I get Ansible to notice a dead target in a timely manner?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` from ``ansible.cfg``. Without this option, SSH and therefore Ansible will wait until the TCP connection times out. Another solution is to add ``ServerAliveInterval`` into your global SSH configuration. A good value for ``ServerAliveInterval`` is up to you to decide; keep in mind that ``ServerAliveCountMax=3`` is the SSH default so any value you set will be tripled before terminating the SSH session.
+You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` from ``ansible.cfg``. Without this option,
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:103]: added formatting code backtick and line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 10]
```diff
@@ -84,32 +83,35 @@ You can create `group_vars/gatewayed.yml` with the following contents::
...
+SSH and therefore Ansible will wait until the TCP connection times out. Another solution is to add ``ServerAliveInterval``
+into your global SSH configuration. A good value for ``ServerAliveInterval`` is up to you to decide; keep in mind that
+``ServerAliveCountMax=3`` is the SSH default so any value you set will be tripled before terminating the SSH session.
 
-.. _ec2_cloud_performance:
+.. _cloud_provider_performance:
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:108]: Change EC2 -> Cloud provider (more generic)

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 11]
```diff
@@ -136,12 +138,12 @@ workaround you can install Python 2 on the managed host and configure Ansible to
 requires Python 2, you can also report a bug on our `bug tracker
 <https://github.com/ansible/ansible/issues>`_ so that the incompatibility can be fixed in a future release.
 
-Do not replace the shebang lines of your python modules.  Ansible will do this for you automatically at deploy time.
+Do not replace the shebang lines of your python modules. Ansible will do this for you automatically at deploy time.
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:141]: Double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 12]
```diff
@@ -136,12 +138,12 @@ workaround you can install Python 2 on the managed host and configure Ansible to
...
 
-Also, this works for ANY interpreter, i.e ruby: `ansible_ruby_interpreter`, perl: `ansible_perl_interpreter`, etc,
+Also, this works for ANY interpreter, i.e ruby: ``ansible_ruby_interpreter``, perl: ``ansible_perl_interpreter``, etc,
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:143]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 13]
```diff
@@ -136,12 +138,12 @@ workaround you can install Python 2 on the managed host and configure Ansible to
...
 so you can use this for custom modules written in any scripting language and control the interpreter location.
 
-Keep in mind that if you put `env` in your module shebang line (`#!/usr/bin/env <other>`),
+Keep in mind that if you put ``env`` in your module shebang line (``#!/usr/bin/env <other>``),
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:146]: added formatting code backtick

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 14]
```diff
@@ -153,7 +155,8 @@ While installing Ansible, sometimes you may encounter errors such as `No package
 These errors are generally caused by the missing packages, which are dependencies of the packages required by Ansible.
 For example, `libffi` package is dependency of `pynacl` and `paramiko` (Ansible -> paramiko -> pynacl -> libffi).
 
-In order to solve these kinds of dependency issues, you might need to install required packages using the OS native package managers, such as `yum`, `dnf`, or `apt`, or as mentioned in the package installation guide.
+In order to solve these kinds of dependency issues, you might need to install required packages using
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:158]: line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 15]
```diff
@@ -195,7 +198,7 @@ need to install them into the virtualenv. There are two methods:
 
       $ virtualenv ansible --system-site-packages
 
-* Copy those files in manually from the system.  For instance, for SELinux bindings you might do:
+* Copy those files in manually from the system. For instance, for SELinux bindings you might do:
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:201]: Double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 16]
```diff
@@ -217,8 +220,9 @@ By default, Solaris 10 and earlier run a non-POSIX shell which does not correctl
 tmp directory Ansible uses ( :file:`~/.ansible/tmp`). If you see module failures on Solaris machines, this
 is likely the problem. There are several workarounds:
 
-* You can set ``remote_tmp`` to a path that will expand correctly with the shell you are using (see the plugin documentation for :ref:`C shell<csh_shell>`, :ref:`fish shell<fish_shell>`, and :ref:`Powershell<powershell_shell>`).  For
-  example, in the ansible config file you can set::
+* You can set ``remote_tmp`` to a path that will expand correctly with the shell you are using
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:223]: Line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 17]
```diff
@@ -241,7 +245,7 @@ There are a few common errors that one might run into when trying to execute Ans
 
 * Version 2.7.6 of python for z/OS will not work with Ansible because it represents strings internally as EBCDIC.
 
-  To get around this limitation, download and install a later version of `python for z/OS <https://www.rocketsoftware.com/zos-open-source>`_ (2.7.13 or 3.6.1) that represents strings internally as ASCII.  Version 2.7.13 is verified to work.
+  To get around this limitation, download and install a later version of `python for z/OS <https://www.rocketsoftware.com/zos-open-source>`_ (2.7.13 or 3.6.1) that represents strings internally as ASCII. Version 2.7.13 is verified to work.
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:248]: Double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 18]
```diff
@@ -264,7 +268,7 @@ There are a few common errors that one might run into when trying to execute Ans
   .. error::
     EE3501S The module libpython2.7.so was not found.
 
-  On z/OS, you must execute python from gnu bash.  If gnu bash is installed at ``/usr/lpp/bash``, you can fix this in your inventory by specifying an ``ansible_shell_executable``::
+  On z/OS, you must execute python from gnu bash. If gnu bash is installed at ``/usr/lpp/bash``, you can fix this in your inventory by specifying an ``ansible_shell_executable``::
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:271]: Double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 19]
```diff
@@ -274,7 +278,7 @@ There are a few common errors that one might run into when trying to execute Ans
 What is the best way to make content reusable/redistributable?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-If you have not done so already, read all about "Roles" in the playbooks documentation.  This helps you make playbook content
+If you have not done so already, read all about "Roles" in the playbooks documentation. This helps you make playbook content
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:281]: Double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 20]
```diff
@@ -304,13 +309,17 @@ that you would like to work in a professional cow-free environment, you can eith
 How do I see a list of all of the ansible\_ variables?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Ansible by default gathers "facts" about the machines under management, and these facts can be accessed in Playbooks and in templates. To see a list of all of the facts that are available about a machine, you can run the "setup" module as an ad-hoc action:
+Ansible by default gathers "facts" about the machines under management, and these facts can be accessed in playbooks
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:312]: added formatting code backtick and line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 21]
```diff
@@ -292,8 +296,9 @@ See :ref:`intro_configuration`.
 How do I disable cowsay?
 ++++++++++++++++++++++++
 
-If cowsay is installed, Ansible takes it upon itself to make your day happier when running playbooks.  If you decide
-that you would like to work in a professional cow-free environment, you can either uninstall cowsay, set ``nocows=1`` in ansible.cfg, or set the :envvar:`ANSIBLE_NOCOWS` environment variable:
+If cowsay is installed, Ansible takes it upon itself to make your day happier when running playbooks. If you decide
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:299]: Line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 22]
```diff
@@ -551,34 +563,31 @@ risky because the parameters and values passed to ``usermod_args`` could
...
 
 .. _web_interface:
 
 Is there a web interface / REST API / etc?
 ++++++++++++++++++++++++++++++++++++++++++
 
-Yes!  Ansible, Inc makes a great product that makes Ansible even more powerful and easy to use. See :ref:`ansible_tower`.
-
-
-.. _docs_contributions:
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:576]: Moved that part downside

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 23]
```diff
@@ -551,34 +563,31 @@ risky because the parameters and values passed to ``usermod_args`` could
...
+  with variables (this will also disable the original warning)
 
 
 .. _commercial_support:
 
 Can I get training on Ansible?
 ++++++++++++++++++++++++++++++
 
-Yes!  See our `services page <https://www.ansible.com/products/consulting>`_ for information on our services and training offerings. Email `info@ansible.com <mailto:info@ansible.com>`_ for further details.
+Yes! See our `services page <https://www.ansible.com/products/consulting>`_ for information on our services
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:578]: Added line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 24]
```diff
@@ -551,34 +563,31 @@ risky because the parameters and values passed to ``usermod_args`` could
 be overwritten by malicious values in the ``host facts`` on a compromised
 target machine. To mitigate this risk:
 
-* set bulk variables at a level of precedence greater than ``host facts`` in the order of precedence found in :ref:`ansible_variable_precedence` (the example above is safe because play vars take precedence over facts)
-* disable the :ref:`inject_facts_as_vars` configuration setting to prevent fact values from colliding with variables (this will also disable the original warning)
+* set bulk variables at a level of precedence greater than ``host facts`` in the order of precedence
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:566]: Added line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 25]
```diff
@@ -468,15 +484,15 @@ Ansible ad-hoc command is the easiest option:
...
+The ``mkpasswd`` utility that is available on most Linux systems is also a great option:
 
 .. code-block:: shell-session
 
     mkpasswd --method=sha-512
 
 
 If this utility is not installed on your system (e.g. you are using macOS) then you can still easily
-generate these passwords using Python. First, ensure that the `Passlib <https://bitbucket.org/ecollins/passlib/wiki/Home>`_
+generate these passwords using Python. First, ensure that the `Passlib <https://foss.heptapod.net/python-libs/passlib/-/wikis/home>`_
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:495]: Updated URL

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 26]
```diff
@@ -447,15 +460,18 @@ For example, to access the value of the HOME environment variable on the managem
...
 
 .. code-block:: jinja
 
-   {{ ansible_env.SOME_VARIABLE }}
+   {{ ansible_env.HOME }}
 
-If you need to set environment variables for TASK execution, see :ref:`playbooks_environment` in the :ref:`Advanced Playbooks <playbooks_special_topics>` section.
-There are several ways to set environment variables on your target machines. You can use the :ref:`template <template_module>`, :ref:`replace <replace_module>`, or :ref:`lineinfile <lineinfile_module>` modules to introduce environment variables into files.
-The exact files to edit vary depending on your OS and distribution and local configuration.
+If you need to set environment variables for TASK execution, see :ref:`playbooks_environment`
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:469]: Added line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 27]
```diff
@@ -304,13 +309,17 @@ that you would like to work in a professional cow-free environment, you can eith
...
+Ansible by default gathers "facts" about the machines under management, and these facts can be accessed in playbooks
+and in templates. To see a list of all of the facts that are available about a machine, you can run the ``setup`` module
+as an ad-hoc action:
 
 .. code-block:: shell-session
 
     ansible -m setup hostname
 
-This will print out a dictionary of all of the facts that are available for that particular host. You might want to pipe the output to a pager.This does NOT include inventory variables or internal 'magic' variables. See the next question if you need more than just 'facts'.
+This will print out a dictionary of all of the facts that are available for that particular host. You might want to pipe
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:320]: added line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 28]
```diff
@@ -353,7 +362,8 @@ file with a list of servers. To do this, you can just access the "$groups" dicti
         {{ host }}
     {% endfor %}
 
-If you need to access facts about these hosts, for instance, the IP address of each hostname, you need to make sure that the facts have been populated. For example, make sure you have a play that talks to db_servers::
+If you need to access facts about these hosts, for instance, the IP address of each hostname,
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:365]: Added Line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 29]
```diff
@@ -373,13 +383,13 @@ How do I access a variable name programmatically?
 +++++++++++++++++++++++++++++++++++++++++++++++++
 
 An example may come up where we need to get the ipv4 address of an arbitrary interface, where the interface to be used may be supplied
-via a role parameter or other input.  Variable names can be built by adding strings together, like so:
+via a role parameter or other input. Variable names can be built by adding strings together, like so:
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:386]: stripped double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 30]
```diff
@@ -373,13 +383,13 @@ How do I access a variable name programmatically?
...
 
 .. code-block:: jinja
 
     {{ hostvars[inventory_hostname]['ansible_' + which_interface]['ipv4']['address'] }}
 
-The trick about going through hostvars is necessary because it's a dictionary of the entire namespace of variables.  ``inventory_hostname``
+The trick about going through hostvars is necessary because it's a dictionary of the entire namespace of variables. ``inventory_hostname``
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:392]: stripped double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 31]
```diff
@@ -396,7 +406,8 @@ Also see dynamic_variables_.
 How do I access a group variable?
 +++++++++++++++++++++++++++++++++
 
-Technically, you don't, Ansible does not really use groups directly. Groups are labels for host selection and a way to bulk assign variables, they are not a first class entity, Ansible only cares about Hosts and Tasks.
+Technically, you don't, Ansible does not really use groups directly. Groups are labels for host selection and a way to bulk assign variables,
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:409]: Added line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 32]
```diff
@@ -406,9 +417,9 @@ That said, you could just access the variable by selecting a host that is part o
 How do I access a variable of the first host in a group?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-What happens if we want the ip address of the first webserver in the webservers group?  Well, we can do that too.  Note that if we
+What happens if we want the ip address of the first webserver in the webservers group?  Well, we can do that too. Note that if we
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:420]: stripped double space

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 33]
```diff
@@ -431,14 +442,16 @@ Notice how we interchanged the bracket syntax for dots -- that can be done anywh
 How do I copy files recursively onto a target host?
 +++++++++++++++++++++++++++++++++++++++++++++++++++
 
-The "copy" module has a recursive parameter.  However, take a look at the "synchronize" module if you want to do something more efficient for a large number of files.  The "synchronize" module wraps rsync.  See the module index for info on both of these modules.
+The ``copy`` module has a recursive parameter. However, take a look at the ``synchronize`` module if you want to do something more efficient
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:445]: added formatting code backtick and line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 34]
```diff
@@ -431,14 +442,16 @@ Notice how we interchanged the bracket syntax for dots -- that can be done anywh
...
+for a large number of files. The ``synchronize`` module wraps rsync. See the module index for info on both of these modules.
 
 .. _shell_env:
 
 How do I access shell environment variables?
 ++++++++++++++++++++++++++++++++++++++++++++
 
-If you just need to access existing variables ON THE CONTROLLER, use the 'env' lookup plugin.
+
+**On controller machine :** Access existing variables from controller use the ``env`` lookup plugin.
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:454]: Put the important information first and emphasize it

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 35]
```diff
@@ -447,15 +460,18 @@ For example, to access the value of the HOME environment variable on the managem
         local_home: "{{ lookup('env','HOME') }}"
 
 
-For environment variables on the TARGET machines, they are available via facts in the 'ansible_env' variable:
+**On target machines :** Environment variables are available via facts in the ``ansible_env`` variable:
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:463]: Put the important information first and emphasize it

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 36]
```diff
@@ -447,15 +460,18 @@ For example, to access the value of the HOME environment variable on the managem
...
 
-   {{ ansible_env.SOME_VARIABLE }}
+   {{ ansible_env.HOME }}
 
-If you need to set environment variables for TASK execution, see :ref:`playbooks_environment` in the :ref:`Advanced Playbooks <playbooks_special_topics>` section.
-There are several ways to set environment variables on your target machines. You can use the :ref:`template <template_module>`, :ref:`replace <replace_module>`, or :ref:`lineinfile <lineinfile_module>` modules to introduce environment variables into files.
-The exact files to edit vary depending on your OS and distribution and local configuration.
+If you need to set environment variables for TASK execution, see :ref:`playbooks_environment`
+in the :ref:`Advanced Playbooks <playbooks_special_topics>` section.
+There are several ways to set environment variables on your target machines. You can use the
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst:471]: Added line break

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 37]
```diff
@@ -641,10 +651,11 @@ For 'non host vars' you can use the :ref:`vars lookup<vars_lookup>` plugin:
 
 .. _why_no_wheel:
 
-Why don't you ship in X format?
-+++++++++++++++++++++++++++++++
+Why don't you ship ansible in wheel format (or other format) ?
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst]: Added back Wheel (as example) and added a more generic case after ("X format" is not obvious what is the topic

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 38]
```diff
@@ -665,7 +676,8 @@ This works for all overridden connection variables, like ``ansible_user``, ``ans
 How do I fix 'protocol error: filename does not match request' when fetching a file?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Newer releases of OpenSSH have a `bug <https://bugzilla.mindrot.org/show_bug.cgi?id=2966>`_ in the SCP client that can trigger this error on the Ansible controller when using SCP as the file transfer mechanism::
+Newer releases of OpenSSH have a `bug <https://bugzilla.mindrot.org/show_bug.cgi?id=2966>`_ in the SCP client
```
Comment (bmillemathias) [inline docs/docsite/rst/reference_appendices/faq.rst]: added line break

Comment (bmillemathias) [reply docs/docsite/rst/reference_appendices/faq.rst:679]: Newer is so unclear, newer than what ? time of writing ? the reader does not know when it was. time of reading ? will it still be true? 
I wish people writing docs gives clearer information, like "starting release 3.04 ..."

**Comment by acozine** [comment]
Thanks @bmillemathias for all this work! Moving forward, we will likely strip out line breaks, because they make translating the documentation more difficult. For now, though, the FAQ page already has a lot of extra line breaks, so I don't mind merging a few more. We can fix them all at once.

**Pull request merged**